appveyor.yml

```diff
@@ -43,6 +43,7 @@ test_script:
   - npm --version
   - make sanity-checks
   - cmd: npm test
+  - cmd: make installers-all
```

Comment:
Just out of curiosity, why is there a `cmd: ` in front of `make installers-all` but not in front of `make sanity-checks` ?